app/src/main/java/com/example/happymeals/database/DatabaseListener.java

```diff
@@ -23,4 +23,6 @@ public interface DatabaseListener {
      * @param listOfSpinners The {@link ArrayList} which holds all the spinner from the DB.
      */
     <T> void onSpinnerFetchSuccess(T listOfSpinners );
+
+    void onSpinnerFetchSuccess(Map<String, Object> data);
```

Comment:
Is this replacing the previous function? If so you can remove the function, if not then what is this one doing differently? Why do we need both?